service-worker.js

```diff
@@ -0,0 +1,22 @@
+---
+---
+
+importScripts('/scripts/sw-toolbox.js');
+
+const VERSION = 1;
```

Comment:
Yes sure. We could use the `site.date` from Jekyll. But this way the entier cache will be invalided even if the content hasn't changed.